prac_09/guitar.py

```diff
@@ -0,0 +1,38 @@
+"""
+CP1404/CP5632 Practical - Suggested Solution
+Guitar class
+"""
+# Note: It would be better to get the current year from the system clock
+# and you might like to do that: lookup the "datetime" module
+CURRENT_YEAR = 2024
```

Comment:
Try the following instead of using 2024

from datetime import datetime

CURRENT_YEAR = datetime.now().year